index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div class="wrapper">
+    <nav class="menu">
+      <div class="logo">
+        <h3>Welcome</h3>
+      </div>
+      <div class="hamburger">
+        <i class="fa-solid fa-bars"></i>
+      </div>
+
+    </nav>
+    <section id="headline">
+      <header>
+        <h2 class="darkColor">Hey there. I'm Gabriel</h2>
+        <h2 class="greenColor">I'm a software developer</h2>
+      </header>
```

Comment:
- For semantics reasons, please remove the `<header>` tag that wraps these two headings and Kindly put the text in line 26 inside a `<h1>` tag instead of `<h2>`.

Reply:
I want the two headlines to be together for the layout with flexBox, instead of header should it would be ok to use  <hgroup> instead?

Reply:
Hi @Omar-Muhamad , the changes I was asked are already part of the last commit. Thanks for the feedback.